src/inspirehep-search-js/templates/literature/results.html

```diff
@@ -18,22 +18,39 @@ <h4 class="custom-h">
                 </h4>
                 <div class="brief-record-details">
                   <div class="authors">
-                    <!-- {{ render_record_authors(record, is_brief=true, show_affiliations=false) }} -->
-                    <span id="record-date">
-                      {{ record.display.date }}
-                    </span>
+                  <span ng-bind-html='record.metadata.collaboration | collaboration : record.metadata.corporate_author'></span>
+                  <span ng-bind-html='record.metadata.authors | authors : record.metadata.collaboration : record.metadata.corporate_author : record.display.number_of_authors : record.metadata.control_number'></span>
+                  <span id="record-date">
+                    - {{ record.display.date }}
+                  </span>
                   </div>
                   <div class="row">
-                    <div class="col-md-12 ">
-                      Journal info goes here
+                    <div ng-if="record.display.publication_info_line.pub_info &amp;&amp; record.metadata.dois"  class="col-md-12">
```

Comment:
to tell you the truth it is working, but it appears like that and I didn't like it .
<img width="128" alt="screen shot 2016-03-16 at 13 59 17" src="https://cloud.githubusercontent.com/assets/7301890/13812995/5812c97e-eb7f-11e5-8a87-73b9a9409a94.png">
I did like this only for the beauty. Should I revert it ?